plutus-core/cost-model/budgeting-bench/Bench.hs

```diff
@@ -4,59 +4,74 @@
 -- See Note [Creation of the Cost Model]
 module Main (main) where
 
-import           PlutusCore                               as PLC
-import qualified PlutusCore.DataFilePaths                 as DFP
+import           CriterionExtensions                             (criterionMainWith)
+import           Nops
+import           PlutusCore                                      as PLC
+import qualified PlutusCore.DataFilePaths                        as DFP
 import           PlutusCore.Evaluation.Machine.ExMemory
+import           PlutusCore.Evaluation.Machine.MachineParameters
 import           PlutusCore.MkPlc
-import           UntypedPlutusCore                        as UPLC
+import           PlutusCore.Pretty                               (Pretty)
+import           UntypedPlutusCore                               as UPLC
 import           UntypedPlutusCore.Evaluation.Machine.Cek
 
+import           Control.DeepSeq                                 (NFData)
 import           Criterion.Main
-import qualified Criterion.Types                          as C
-import qualified Data.ByteString                          as BS
+import           Criterion.Types                                 as C
+import qualified Data.ByteString                                 as BS
 import           Data.Functor
-import qualified Hedgehog                                 as HH
-import qualified Hedgehog.Internal.Gen                    as HH
-import qualified Hedgehog.Internal.Tree                   as HH
-import qualified Hedgehog.Range                           as HH.Range
+import           Data.Ix                                         (Ix)
+import           Data.Typeable                                   (Typeable)
+import qualified Hedgehog                                        as HH
+import qualified Hedgehog.Internal.Gen                           as HH
+import qualified Hedgehog.Internal.Tree                          as HH
+import qualified Hedgehog.Range                                  as HH.Range
 import           System.Directory
-import           System.Random                            (StdGen, getStdGen, randomR)
+import           System.Random                                   (StdGen, getStdGen, randomR)
 
-type PlainTerm = UPLC.Term Name DefaultUni DefaultFun ()
+type PlainTerm fun = UPLC.Term Name DefaultUni fun ()
 
 -- TODO.  I'm not totally sure what's going on here.  `env` is supposed to
 -- produce data that will be supplied to the things being benchmarked.  Here
 -- we've got a term and we evaluate it to get back the budget consumed, but then
 -- we throw that away and evaluate the term again.  This may have the effect of
 -- avoiding warmup, which could be a good thing.  Let's look into that.
-runTermBench :: String -> PlainTerm -> Benchmark
-runTermBench name term = env
+-- benchWith :: Int -> String -> PlainTerm fun -> Benchmark
```

Comment:
commented out

Reply:
Well spotted: it got swallowed up by the adjoining comment.